### DISCUSSION TITLE
Reorganize with a start.sh in the root and everything else in a /source or /lib folder?

### DISCUSSION THREAD

**Discussion by synth-mania** [TheBlewish/Automated-AI-Web-Researcher-Ollama#26] [Ideas]
Just for ease of use, what do you think about having a "start.sh" in the root with almost everything else in a subdirectory?

**Comment by TheBlewish** [comment]
Thanks for the idea, I think I will implement something like that, I just have like 2 University exams today, but I'm definitely going to look at implementing that asap

**Comment by synth-mania** [comment]
Cool. I might send a pull request your way if I have time.

**Comment by TheBlewish** [comment]
If you could send the same thing without the extra APIs which I explained why in the pull request I declined, that'd be great! 
Otherwise no worries thanks!

**Reply by synth-mania** [reply]
Sure, give me a little bit, and I'll reorganize based on your main branch and PR that
